src/k8s-extension/README.rst

```diff
@@ -31,6 +31,8 @@ az k8s-extension create \
     --version versionNumber \
     --auto-upgrade-minor-version autoUpgrade \
     --configuration-settings exampleSetting=exampleValue \
+    --plan-info name=examplePlanName publisher=examplePublisher product=exampleOfferId \
+
```

Comment:
Should this be updated?

As per my understanding in previous meeting, we are going with VM and ManagedApps model for plan, i.e., having separate args for name, publisher and product

Reply:
thank you. i forgot to update it at this point. Done now 👍